### PR TITLE
Endpoint to delete CVs using the parameter ID

### PR DESCRIPTION
**Pull request by CharlesKimMaina**
# Description
This is the Endpoint from where you can delete Cvs using ID parameter

Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/3

# How to test?
On postman, i have tested it and results are as below;
delete_cv_byId

# Checklist

- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/cvs.controller.js [hunk 1]
```diff
@@ -48,10 +42,8 @@ const editCv = async (cvId, updatedCv) => {
     });
 };
 
-const deleteCv = async (cvId) => {
-  return knex('cv')
-    .where({ id: cvId })
-    .del();
+const deleteCv = async (cvsId) => {
+  return knex('cvs').where({ id: cvsId }).del();
 };
```
Comment (DiegoRuiz454) [inline src/server/api/controllers/cvs.controller.js:47]: good job charles. this is a clean code!! i have tested and everything works good. cheers!

File: src/server/api/routes/cvs.router.js [hunk 2]
```diff
@@ -147,13 +144,13 @@ router.patch('/:id', (req, res, next) => {
...
  *      200:
- *        description:  CV deleted
+ *        description:  CV is deleted
  *      5XX:
  *        description: Unexpected error.
  */
 router.delete('/:id', (req, res) => {
   cvsController
-    .deleteModule(req.params.id, req)
+    .deleteCv(req.params.id, req)
```
Comment (DiegoRuiz454) [inline src/server/api/routes/cvs.router.js:153]: Approved ready to merge into production !!

**Review by DiegoRuiz454 — APPROVED** [review]
The DELETE endpoint api/cvs/{id} have been tested and is ready to merge.
delete

File: src/server/api/controllers/cvs.controller.js (hunk 1 above)
Comment (CharlesKimMaina) [reply src/server/api/controllers/cvs.controller.js:47]: Thanks Diego,.,. cheers

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Hi @CharlesKimMaina, for some reason a lot of style changes is introduced in this PR. Can i have you remove those? I.e if you look at "files changed" we see too many changes in cvs.controller.js. 

Otherwise, it looks good.

**Pull request closed without merging**